lib/src/manager/hms_sdk_manager.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:hmssdk_flutter/src/service/platform_service.dart';
+
+import '../../hmssdk_flutter.dart';
+
+class HmsSdkManager {
+  Future<bool> createInstance(HMSTrackSetting? hmsTrackSetting) async {
+    bool isCreated = await createHMSSdk(hmsTrackSetting);
+    return isCreated;
+  }
+
+  Future<bool> createHMSSdk(HMSTrackSetting? hmsTrackSetting) async {
+    return await PlatformService.invokeMethod(PlatformMethod.createSdk,
```

Comment:
@jashan1498 this should be `build` function. "createSdk" should be renamed to "build"